### PR TITLE
New footer

### PR DESCRIPTION
**Pull request by mbouzi**
- update footer
- fix mobile navbar

### CODE REVIEW

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -1,120 +1,82 @@
...
+				<!-- <li><a class="footer__link" href="#">Apps</a></li> -->
+				<li><a class="footer__link" href="https://tardigrade.io/community/">Community</a></li>
+				<li><a class="footer__link" href="tardigrade.io/tech-partner/">Partners</a></li>
+				<li><a class="footer__link" href="https://support.tardigrade.io/hc/en-us">Support</a></li>
+	    </ul>
+	  </div>
+	  <div class="footer__content-item col-6 col-sm-6 col-md-6 col-lg-1">
+	    <h5 class="text-white">Company</h5>
+	    <ul class="list-unstyled">
+				<li><a class="footer__link" href="http://tardigrade.io/blog">Blog</a></li>
```
Comment (VitaliiShpital) [inline layouts/partials/footer.html]: same about https

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -1,120 +1,82 @@
...
+	  <div class="footer__content-item footer__content-item-btn offset-lg-1 col-6 col-md-6 col-lg-3">
+	    <h5 class="text-white">Stay in Touch</h5>
+			<p class="footer-message">For the best experience, disable ad blockers and use Chrome or Edge.</p>
+	    <div id="mc_embed_signup">
+	      <form method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+	        <div id="mc_embed_signup_scroll">
+	          <input class="footer__email-input" type="email" value="" name="EMAIL" id="mce-EMAIL-newsletter" placeholder="Email Address" required="">
+	          <input class="footer__btn-submit btn-signup" type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe-footer">
+	            <div id="mce-responses" class="clear">
+	              <div class="response" id="mce-error-response-newsletter" style="display:none;">The email address you provided is not valid!</div>
```
Comment (VitaliiShpital) [inline layouts/partials/footer.html]: we should avoid inline styles

File: static/img/footer.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+					<h5 class="footer-title">Legal</h5>
+          <ul class="list-unstyled">
+            <li><a class="footer-link" href="#">Privacy Policy</a></li>
+            <li><a class="footer-link" href="#">Terms of Service</a></li>
+          </ul>
+        </div>
+        <div class="col-6 col-sm-6 col-md-3 col-lg-2">
+          <h5 class="footer-title">Resources</h5>
+          <ul class="list-unstyled">
+						<li><a class="footer-link" href="http://documentation.tardigrade.io/">Docs</a></li>
```
Comment (VitaliiShpital) [inline static/img/footer.html:19]: https

File: static/img/footer.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+						<!-- <li><a class="footer-link" href="#">Apps</a></li> -->
+						<li><a class="footer-link" href="https://tardigrade.io/community/">Community</a></li>
+						<li><a class="footer-link" href="tardigrade.io/tech-partner/">Partners</a></li>
+						<li><a class="footer-link" href="https://support.tardigrade.io/hc/en-us">Support</a></li>
+          </ul>
+        </div>
+        <div class="col-6 col-sm-6 col-md-2 col-lg-2">
+          <h5 class="footer-title">Company</h5>
+          <ul class="list-unstyled">
+						<li><a class="footer-link" href="http://tardigrade.io/blog">Blog</a></li>
```
Comment (VitaliiShpital) [inline static/img/footer.html:29]: same

File: assets/scss/_fonts.scss [hunk 5]
```diff
@@ -1 +1,2 @@
 $primary-font: Poppins;
+$secondary-font: Poppins;
```
Comment (VitaliiShpital) [inline assets/scss/_fonts.scss]: what's the difference between this two variables? I mean, primary and secondary?

Comment (mbouzi) [reply assets/scss/_fonts.scss]: apologies was supposed to be Roboto

**Pull request merged**